examples/deploy_ecs/defs.py

```diff
@@ -17,6 +17,4 @@ def my_graph():
 my_job = my_graph.to_job()
 
 
```

Comment:
Not directly related, but while we're ceaning this up, I think this could probably be

```
@job
def my_job():
  my_op()
```

since turning graphs into jobs is kind of an advanced feature

Reply:
It makes sense! Done in [b1e28ca](https://github.com/dagster-io/dagster/pull/19038/commits/b1e28ca8a287e5719ac9807920d88001bc40de20)